src/plugins/workspace/public/components/workspace_detail/workspace_detail.test.tsx

```diff
@@ -119,6 +119,11 @@ const WorkspaceDetailPage = (props: any) => {
         },
       },
       dataSourceManagement,
+      navigationUI: {
+        HeaderControl: ({ controls }) => {
+          return controls?.[0].testId ?? null;
```

Comment:
```suggestion
          return controls.map(control => <button data-test-subj={control.testId} onClick={control.run}>{control.content}</button>)
```
The main point to fake this component is to ensure we have 100% test coverage on the delete modal.